test/Base_Tests/polyglot-sources/enso-test-java-helpers/src/main/java/org/enso/base_test_helpers/ExecutableLocation.java

```diff
@@ -55,7 +55,12 @@ private static File getExecutableLocationFromModule() {
     }
     File exec;
     if (isOnWindows()) {
-      exec = binDir.resolve("enso.bat").toFile();
+      var exe = binDir.resolve("enso.exe").toFile();
+      if (exe.canExecute()) {
```

Comment:
Why `canExecute` instead of `exists`?

I imagine that if the file exists, it should be executable. `enso.exe` existing but not executable would be some invalid exceptional state.

Or am I missing something?

Reply:
`canExecute` implies `exists`, and `canExecute` is checked in the end. I thought it is reasonable to check it this way https://github.com/enso-org/enso/blob/e41cb2f5ace1049feb44a6f1ad738184ace523d7/test/Base_Tests/polyglot-sources/enso-test-java-helpers/src/main/java/org/enso/base_test_helpers/ExecutableLocation.java#L57-L63

Reply:
Ok